src/main/resources/assets/js/app/view/context/widget/version/VersionsView.ts

```diff
@@ -88,136 +95,217 @@ export class VersionsView
+    }
+
+    private initListeners() {
+        this.addOnClickHandler();
+        this.addCompareButtonAction();
+    }
+
+    private addOnClickHandler() {
+        this.onClicked(() => {
+            this.collapseAllContentVersionItemViewsExcept(this);
+
+            if (!this.hasClass('active') || this.hasClass('online')) {
+                this.toggleClass('expanded');
+            }
+        });
+    }
 
-        return descriptionDiv;
+    private collapseAllContentVersionItemViewsExcept(itemContainer: Element) {
+        $(this.getHTMLElement()).find('.content-version-item').not(itemContainer.getHTMLElement()).removeClass('expanded');
     }
 
-    private createVersionInfoBlock(item: ContentVersion): Element {
-        const versionInfoDiv = new DivEl('version-info hidden');
+    private addCompareButtonAction() {
+        this.compareButton.getAction().onExecuted(() => {
+            CompareContentVersionsDialog.get()
+                .setContent(this.content.getContentSummary())
+                .setLeftVersion(this.contentVersion.id)
+                .setActiveVersion(this.activeVersion.id)
+                .open();
+        });
+    }
 
-        if (item.publishInfo) {
-            if (item.publishInfo.message) {
-                const messageDiv = new DivEl('version-info-message');
-                messageDiv.appendChildren(new PEl('message').setHtml(item.publishInfo.message));
-                versionInfoDiv.appendChild(messageDiv);
+    doRender(): Q.Promise<boolean> {
+        return super.doRender().then((rendered) => {
+            if (this.statusBlock) {
+                const statusClass: string = this.getStatusClass();
+                this.statusBlock.addClass(statusClass);
+                this.addClass(statusClass);
+                this.appendChild(this.statusBlock);
             }
 
-            const publisher = new NamesAndIconViewBuilder().setSize(NamesAndIconViewSize.small).build();
-            publisher
-                .setMainName(item.publishInfo.publisherDisplayName)
-                .setSubName(DateHelper.getModifiedString(item.publishInfo.timestamp))
-                .setIconClass(item.isStateReady() ? 'icon-state-ready' : 'icon-state-in-progress');
+            this.descriptionBlock.addClass('description');
+            this.descriptionBlock.appendChild(this.compareButton);
+
+            this.appendChild(this.descriptionBlock);
+            this.appendChild(this.versionInfoBlock);
+
+            return rendered;
+        });
+    }
+
+    private getStatusClass(): string {
+        if (!this.contentVersion.isInMaster()) {
+            return CompareStatusFormatter.formatStatusClassFromContent(this.content).toLowerCase();
+        }
+
+        if (this.isPublishPending()) {
+            return `${CompareStatusFormatter.formatStatus(CompareStatus.EQUAL, null, true)} ${PublishStatus.PENDING}`.toLowerCase();
+        }
+
+        return CompareStatusFormatter.formatStatus(CompareStatus.EQUAL).toLowerCase();
```

Comment:
This should be 

`CompareStatusFormatter.formatStatus(CompareStatus.EQUAL, null, true).toLowerCase()
`

CSS class is incorrect now:
 
![image](https://user-images.githubusercontent.com/11612490/75262889-7335e900-57ed-11ea-94b9-d46b881eb41c.png)